src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -824,6 +824,20 @@ def load_arguments(self, _):
                    help="swap types. use 'preview' to apply target slot's settings on the source slot first; use 'swap' to complete it; use 'reset' to reset the swap",
                    arg_type=get_enum_type(['swap', 'preview', 'reset']))
 
+    with self.argument_context('functionapp deployment github-actions')as c:
+        c.argument('name', arg_type=functionapp_name_arg_type)
+        c.argument('resource_group', arg_type=resource_group_name_type)
+        c.argument('repo', help='The GitHub repository to which the workflow file will be added. In the format: https://github.com/<owner>/<repository-name> or <owner>/<repository-name>')
+        c.argument('token', help='A Personal Access Token with write access to the specified repository. For more information: https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line')
+        c.argument('slot', options_list=['--slot', '-s'], help='The name of the slot. Default to the production slot if not specified.')
+        c.argument('branch', options_list=['--branch', '-b'], help='The branch to which the workflow file will be added. Defaults to "master" if not specified.')
```

Comment:
We don't need "Defaults to "master" if not specified." here since the help text automatically displays default values: 
<img width="1170" alt="Screen Shot 2022-07-26 at 12 07 10 PM" src="https://user-images.githubusercontent.com/10121214/181091251-d8caf500-9000-4a6b-906d-2824fcad4070.png">

Reply:
fixed